posthog/hogql/functions/mapping.py

```diff
@@ -100,6 +101,58 @@ def compare_types(arg_types: list[ConstantType], sig_arg_types: tuple[ConstantTy
     )
 
 
+# The pattern would capture any of these keywords
+# when they appear in the format KEYWORD(...) or KEYWORD (...) in SQL
+SQL_KEYWORDS = {
```

Comment:
Don't know if you've seen this, but these are the keywords we support: https://github.com/PostHog/posthog/blob/master/posthog/hogql/grammar/HogQLParser.g4#L290-L300 - maybe adding a comment that this list should be kept updated with the grammar keywords would be nice